src/modules/mavlink/mavlink_params.c

```diff
@@ -141,3 +141,25 @@ PARAM_DEFINE_INT32(MAV_FWDEXTSP, 1);
  * @group MAVLink
  */
 PARAM_DEFINE_INT32(MAV_BROADCAST, 0);
+
+/**
+ * Parameter hash check.
+ *
+ * Disabling the parameter hash check functionality will make the mavlink instance
+ * stream parameters continuously.
+ *
+ * @boolean
+ * @group MAVLink
+ */
+PARAM_DEFINE_INT32(MAV_EN_HASH_CHK, 1);
```

Comment:
This is picky, but I'm wondering if it would make slightly more sense to instead add `_EN` as the suffix so that it sorts properly in the list and is more consistent with other boolean parameters?  `MAV_EN_HASH_CHK -> MAV_HASH_CHK_EN`

Examples
CAL_ACCx_EN, VT_WV_LND_EN, VT_FW_DIFTHR_EN, FW_BAT_SCALE_EN, SYS_STCK_EN

Thoughts?